modules/apps/frontend-js/frontend-js-recharts/liferay-npm-bundler.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	"create-jar": false,
```

Comment:
This is unfortunate... hopefully we can hide this default in the presets when we roll this out globally...

Reply:
@jbalsas The problem here is that the portal's build is different from the one of the rest of the world (at least for now) because the portal delegates the JAR assembling to gradle, while the rest of the people usually ask the bundler to create the JAR (except when using blade, of course).

We can hide it in liferay-npm-scripts or add an autodetection feature in the bundler, but I'm not quite sure if that is a good idea, because it looks like the bundler shouldn't care about those things...

Reply:
That's what I mean... it should be somewhere else, not in the bundler, and definitely not in its configuration. We might want to reduce the bundler API surface and then provide something on top composing it when needed.

Reply:
Not in its configuration? How is the bundler going to know if it has to assemble the JAR, then?

Reply:
If we move it to other layer, there's some code in the bundler we would need to move there.

Reply:
Why would the bundler need to know how to assemble a JAR?

Indeed, we will probably want to move it elsewhere... and that might even be blade/workspace in the future.

Reply:
In that case, instead of moving the code I mentioned above, we can generate those files too for the other tool to assemble (we are now generating them in memory when composing the JAR).

However, that would force frontend developers to install a JDK + blade even if they just wanted to create pure JS portlets. The main reason why we decided to create the JAR from the bundler was that: pure JS development without any dependency.

Reply:
I remember the reasons why we decided to do that 😉